### PR TITLE
Update answers on transfer minimums.

### PR DESCRIPTION
**Pull request by jordanmack**
Update answers for:

- Why is it that the minimum transfer amount cannot be less than 61 CKB?
- If I have 100 CKB in my address and transfer 61 CKB, the balance remaining (39 CKB) isn’t enough to create a change cell. How should I handle this?

### CODE REVIEW

File: website/docs/integrate/qa.md [hunk 1]
```diff
@@ -36,19 +36,32 @@ https://github.com/nervosnetwork/ckb-sdk-java/blob/develop/ckb/src/test/java/uti
 https://github.com/nervosnetwork/ckb-sdk-java/blob/develop/ckb/src/test/java/utils/AddressGeneratorTest.java
 
 
-**Q:** Why the minimum transfer amount cannot be less than 61 CKB?
+**Q:** Why is it that the minimum transfer amount cannot be less than 61 CKB?
 
-**A:** A Cell has three fields and itself should take up capacity, you may refer to [Cell](https://nervosnetwork.github.io/docs-new/docs/reference/cell) for more details：
+**A:** A cell is used to represent a balance on-chain, and one must hold 1 CKB for every 1 byte of space that the cell occupies. The data in a every cell is broken down into four fields: capacity, lock script, type script, and data. Capacity and lock script are required, and type script and data are optional. For more detailed information about the structure of a cell, refer to the [cell](https://nervosnetwork.github.io/docs-new/docs/reference/cell) reference page.
```
Comment (xying21) [inline website/docs/integrate/qa.md:41]: The data in **a every** cell

Comment (xying21) [inline website/docs/integrate/qa.md:41]: The data in **a every** cell

File: website/docs/integrate/qa.md [hunk 2]
```diff
@@ -36,19 +36,32 @@ https://github.com/nervosnetwork/ckb-sdk-java/blob/develop/ckb/src/test/java/uti
...
-* The field `capacity` is u64, which takes 8 bits.
-* The field `lock script` is `Script` type, includes `code_hash` 32 bits, `hash_type` 1 bit, `args` 20 bits
-* The field `type script` is optional.
+The most common basic cell is one that represent a CKB balance. Often this will use the default lock script, which is used for short CKB addresses supported by most wallets. A basic cell like this does not require a type script or any additional data. The fields of this cell are as follows:
 
-so the minimum transfer amount need to be 61 CKB.
+* The field `capacity` field is used to hold the number of CKB balance in the cell. This is a u64, which takes 8 bytes.
+* The field `lock script` is `Script` type. This inclues a `code_hash` that is 32 bytes, a `hash_type` that is 1 byte, and an `args` field that is 20 bytes.
+* The field `type script` is optional and is not used for this cell.
+* The field `data` is optional and is not used for this cell.
```
Comment (xying21) [inline website/docs/integrate/qa.md:48]: The `capacity` field 
The `lock script` field 
The `type script` field 
The `data` field

File: website/docs/integrate/qa.md (hunk 1 above)
Comment (jordanmack) [reply website/docs/integrate/qa.md:41]: Should be "every".

File: website/docs/integrate/qa.md (hunk 2 above)
Comment (jordanmack) [reply website/docs/integrate/qa.md:48]: 👍‍

**Pull request merged**